addons/payment/views/payment_token_views.xml

```diff
@@ -5,7 +5,7 @@
         <field name="name">payment.token.form</field>
         <field name="model">payment.token</field>
         <field name="arch" type="xml">
-            <form string="Payment Tokens" create="false" editable="bottom">
+            <form string="Payment Tokens" create="false">
```

Comment:
We must make the form readonly now that the write perm is back.
```suggestion
            <form string="Payment Tokens" create="false" edit="false">
```

Reply:
Tree view as well?

Reply:
Yup (or set create: False in the context of the token action, should work as well AFAIK)

Reply:
Edit: ioups, read too fast, for the tree view it's indeed not necessary since you cannot edit records unless `multi_edit` is specified and truthy.